python/TestHarness/testers/ICEUpdaterTester.py

```diff
@@ -25,24 +25,13 @@ def __init__(self, name=None, params=None):
        self.nPosts = self.getParam('nPosts')
        self.port = int(self.getParam('port'))
 
-    self.httpServer = SimpleHttpServer("localhost", self.port)
-    self.httpServer.start()
-
   # This method is called prior to running the test.  It can be used to cleanup files
   # or do other preparations before the tester is run
   def prepare(self):
+    self.httpServer = SimpleHttpServer("localhost", self.port)
+    self.httpServer.start()
     return
```

Comment:
It's better to start the server here right before the test. When running a large number of tests in parallel there can be a big break between the construction of a job and the running of that job.